packages/suite/src/actions/wallet/selectedAccountActions.ts

```diff
@@ -170,7 +171,13 @@ const actions = [
     blockchainActions.connected.type,
     blockchainActions.reconnectTimeoutStart.type,
     blockchainActions.updateFee.type,
-    ...Object.values(DISCOVERY).filter(v => typeof v === 'string'),
+    discoveryActions.stopDiscovery,
+    discoveryActions.interruptDiscovery,
+    discoveryActions.createDiscovery,
+    discoveryActions.startDiscovery,
+    discoveryActions.updateDiscovery,
+    discoveryActions.removeDiscovery,
+    discoveryActions.completeDiscovery,
```

Comment:
missing `.type`? looking for action.type on #L188

Reply:
[a802d5f](https://github.com/trezor/trezor-suite/pull/8616/commits/a802d5f07e68c6142a2e8f18da68635a240d397e)